buildSrc/src/main/java/Modules.kt

```diff
@@ -0,0 +1,7 @@
+object Modules {
+    const val app = ":app"
+
+    const val core = ":core"
+    const val coreUi = ":core-ui"
+
```

Comment:
Please, fix format and remove unused spaces.

Reply:
Does lint not check/catch these formatting issues?

Reply:
It does. 

I think he hasn't pulled and merged the lint when he raised this PR.